R/transform_log.R

```diff
@@ -0,0 +1,19 @@
+#' transform_log 
+#' Transform numerical values into their log values
+#' @param x A vector
```

Comment:
Parameter description could be more specific here, e.g. indicating the data type.